Documentation/ceph-upgrade.md

```diff
@@ -487,53 +493,20 @@ variables matching `ROOK_CSI_*_IMAGE` from the above ConfigMap and/or the operat
 
 ### Verifying updates
 
-You can use the below command to see the CSI images currently being used in the cluster.
+You can use the below command to see the CSI images currently being used in the cluster. Note that
+not all images (like `volumereplication-operator`) may be present in every cluster depending on
+which CSI features are enabled.
 
 ```console
 kubectl --namespace rook-ceph get pod -o jsonpath='{range .items[*]}{range .spec.containers[*]}{.image}{"\n"}' -l 'app in (csi-rbdplugin,csi-rbdplugin-provisioner,csi-cephfsplugin,csi-cephfsplugin-provisioner)' | sort | uniq
 ```
 
 ```
-quay.io/cephcsi/cephcsi:v3.3.1
 k8s.gcr.io/sig-storage/csi-attacher:v3.2.1
 k8s.gcr.io/sig-storage/csi-node-driver-registrar:v2.2.0
 k8s.gcr.io/sig-storage/csi-provisioner:v2.2.2
 k8s.gcr.io/sig-storage/csi-resizer:v1.2.0
 k8s.gcr.io/sig-storage/csi-snapshotter:v4.1.1
+quay.io/cephcsi/cephcsi:v3.3.1
+quay.io/csiaddons/volumereplication-operator:v0.1.0
 ```
-
-## Replace lvm mode OSDs with raw mode (if you use LV-backed PVC)
```

Comment:
Should we remove this section from the v1.6 upgrade guide as well since raw mode has issues?

Reply:
I suppose so. I can do that separately.

Reply:
Actually, reading this more closely, this is really about PVCs. Maybe we should leave it. @leseb ?

Reply:
I think @satoru-takeuchi might have written that since it's LV-backend PV :). I'm fine removing it for the 1.7 upgrades.